packages/client/src/components/atoms/BookingsCountdown/AddToCalendar.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
```

Comment:
This has nothing to do with `BookingsCountdown` and should be in it's own folder...same as the tests. So do the standard `index.ts`, `<ComponentName>.tsx`,  `__tests__/<ComponentName>.test.tsx` folder structure.